MBBSEmu/HostProcess/ExportedModules/ExportedModuleBase.cs

```diff
@@ -225,11 +225,23 @@ private protected string GetParameterString(int parameterOrdinal, bool stripNull
             return Encoding.ASCII.GetString(Module.Memory.GetString(stringPointer, stripNull));
         }
 
+        /// <summary>
+        ///     Gets a string Parameter as a byte array
+        /// </summary>
+        /// <param name="parameterOrdinal"></param>
+        /// <returns></returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private protected ReadOnlySpan<byte> GetParameterByteArray(int parameterOrdinal, bool stripNull = false)
```

Comment:
I would rename this to `GetParameterStringSpan` as that's actually what it's doing

Reply:
Renamed